QualityControl/public/layout/view/header.js

```diff
@@ -62,7 +62,7 @@ const toolbarViewMode = (model) => {
         }, iconShareBoxed()),
         model.session.personid == layoutItem.owner_id && [
           h('button.btn.btn-primary', {
-            onclick: () => model.layout.edit(),
+            onclick: async () => await model.layout.edit(),
```

Comment:
This is an event trigger, there is no need to await for it as nothing else is done after.
It is enough to just fire the call and forget about it.